packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -326,6 +327,31 @@ export class GaussianSplattingMesh extends Mesh {
         super.dispose(doNotRecurse);
     }
 
+    /**
+     * Returns a new Mesh object generated from the current mesh properties.
+     * @param name is a string, the name given to the new mesh
+     * @returns a new Gaussian Splatting Mesh
+     */
+    public override clone(name: string = ""): GaussianSplattingMesh {
+        const newGS = new GaussianSplattingMesh(name, undefined, this.getScene());
+        newGS._copySource(this);
+        newGS._vertexCount = this._vertexCount;
+        newGS._modelViewMatrix = Matrix.Identity();
+        newGS._covariancesATexture = this._covariancesATexture;

```

Comment:
What is the good practice for sharing these textures across clones? First disposed clone will dispose the textures as well.

Reply:
In materials, we clone the textures, but it is easy as they are not dynamic, and they end up reusing the same internal texture (ref count is done within the textures).

I guess we should do the same and try cloning the texture ?

Reply:
Doesn't work. I have a small PG that repo it : https://playground.babylonjs.com/#Y4N3UT

Reply:
on it

Reply:
https://github.com/BabylonJS/Babylon.js/pull/15299